### PR TITLE
MBS-7958 Add test-instrumentation-runner without avito specific depen…

### PR DESCRIPTION
**Pull request by sergioboi**
…dencies

### CODE REVIEW

File: subprojects/android-test/test-instrumentation-runner/src/main/kotlin/com/avito/android/runner/InstrumentationTestRunner.kt [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+                return DefaultTestInstrumentationContext(
+                    errorsReporter = LogErrorsReporter()
+                )
+            }
+        }
+    }
+
+    override fun onCreate(arguments: Bundle) {
+        val context = createFactory().create(arguments)
+        if (context != null) {
```
Comment (eugene-krivobokov) [inline subprojects/android-test/test-instrumentation-runner/src/main/kotlin/com/avito/android/runner/InstrumentationTestRunner.kt:22]: Should we fail if null?

File: subprojects/android-test/test-instrumentation-runner/src/main/kotlin/com/avito/android/runner/Context.kt [hunk 2]
```diff
@@ -0,0 +1,5 @@
+package com.avito.android.runner
+
+interface Context {
```
Comment (eugene-krivobokov) [inline subprojects/android-test/test-instrumentation-runner/src/main/kotlin/com/avito/android/runner/Context.kt:3]: What is the responsibility of this abstraction?
What will it know besides errors reporter?

File: subprojects/android-test/test-instrumentation-runner/src/main/kotlin/com/avito/android/runner/InstrumentationTestRunner.kt (hunk 1 above)
Comment (sergioboi) [reply subprojects/android-test/test-instrumentation-runner/src/main/kotlin/com/avito/android/runner/InstrumentationTestRunner.kt:22]: It returns null when we in orchestrator execution

File: subprojects/android-test/test-instrumentation-runner/src/main/kotlin/com/avito/android/runner/Context.kt (hunk 2 above)
Comment (sergioboi) [reply subprojects/android-test/test-instrumentation-runner/src/main/kotlin/com/avito/android/runner/Context.kt:3]: It is a service locator for the future. I consider to use it for holding all needed dependencies

**Review by eugene-krivobokov — APPROVED** [review]

**Pull request merged**